app/models/manageiq/providers/nuage/network_manager/event_catcher/stream.rb

```diff
@@ -0,0 +1,45 @@
+require 'qpid_proton'
```

Comment:
the `qpid_proton` gem is nowhere included. AFAIK we dont use it yet in the core.

@kbrock when / will we include it in the cores Gemfile?

If not you'll have to add it to the providers gemspec

Reply:
@gberginc  if you add a spec, that will trigger this code path, I guess it will throw an exception

Reply:
> If not you'll have to add it to the providers gemspec

It should be included in the provider's gemspec regardless, because the provider should always declare it's dependencies.  This way if we, say, have to bump the version in the main repo, we can ensure we pick a version that satisfies the dependencies of this plugin

Reply:
Same...this still cannot be at the top-level context

Reply:
I don't think you even need this as long as the MessagingHandler does the require in it's initialize method.